README.md

```diff
@@ -1,2 +1,2 @@
 # pa-algorithms
-Repository with algorithms for Physical Activity.
\ No newline at end of file
+Repository with Perfect Fit algorithms for physical activities and training plans.
```

Comment:
Could you add some installation instructions? Also see https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch.

Reply:
Hi @svenvanderburg, I added a line with stallation instructions.

Reply:
Cool! Thanks :)